jenkins/pipelines/ci/tikv/tikv_ghpr_test.groovy

```diff
@@ -179,7 +179,9 @@ stage("Prepare") {
                         sh label: 'Run lint: format', script: """
                             cd \$HOME/tikv-src
                             export RUSTFLAGS=-Dwarnings
-                            make format && git diff --quiet || (git diff; echo Please make format and run tests before creating a PR; exit 1)
+                            make format
```

Comment:
why doesn't make format run cargo sort?

Reply:
Oh, I didn't notice it's already in the Makefile. Maybe it's because the CI environment doesn't have cargo sort.

Reply:
pre-format installs cargo-sort automatically, why install it explicitly?